src/Microsoft.AspNetCore.Razor.Tools/Client.cs

```diff
@@ -71,15 +67,6 @@ public static async Task<Client> ConnectAsync(string pipeName, TimeSpan? timeout
                 ServerLogger.Log("Named pipe '{0}' connected", pipeName);
                 cancellationToken.ThrowIfCancellationRequested();
 
-#if NET46
-                // Verify that we own the pipe.
-                if (!CheckPipeConnectionOwnership(stream))
```

Comment:
Actually, I think this might still be an issue. We probably need to continue cross-compiling this for netfx because this is code is needed in netfx.

Reply:
/cc @ajaybhargavb

Reply:
Razor.Tools doesn't cross-compile: https://github.com/aspnet/Razor/blob/master/src/Microsoft.AspNetCore.Razor.Tools/Microsoft.AspNetCore.Razor.Tools.csproj#L6 

It's always run as a .NET Core process

Reply:
> It's always run as a .NET Core process

Not always. This file is shared by both Razor.Tools and NET.Sdk.Razor https://github.com/aspnet/Razor/blob/master/src/Microsoft.NET.Sdk.Razor/Microsoft.NET.Sdk.Razor.csproj#L33. So I agree with @rynowak here.